neonvm/runner/main.go

```diff
@@ -881,13 +881,22 @@ func forwardLogs(ctx context.Context, logger *zap.Logger, wg *sync.WaitGroup) {
 	delay := 3 * time.Second
 	var conn net.Conn
 	var reader *bufio.Reader
+
 	b := &backoff.Backoff{
 		Min:    100 * time.Millisecond,
 		Max:    delay,
 		Factor: 2,
 		Jitter: true,
 	}
 
+	// Wait a bit to reduce the chance we attempt dialing before
+	// QEMU is started
+	select {
+	case <-time.After(500 * time.Millisecond):
```

Comment:
Adding sleeps seems a bit hacky imo, especially one as long as 0.5s. Can we get away with not having this?

Reply:
Yep, it is a bit hacky. Probably the idiomatic solution would be to have inotify, but then this sleep is not critical, since we have a backoff below anyway. So I decided not to bother. 

I can shorten it to 200ms, feels like enough for QEMU to create the socket. WDYT?

Reply:
IIUC without the wait it _will_ work, but it'll have unrelated errors, right? 200ms should be fine - boot takes 0.4s in prod I think? As long as we aren't risking delaying compute_ctl startup by blocking its log sending, we should be good I think

Reply:
> IIUC without the wait it will work, but it'll have unrelated errors, right? 

Yes

> 200ms should be fine - boot takes 0.4s in prod I think? 

Yes